### ISSUE TITLE
Separate "Reset leases" button for DHCP leases (non-static)

### ISSUE THREAD

**Issue by UltimateNova1203** [AdguardTeam/AdGuardHome#6804]
### Prerequisites

- [X] I have checked the [Wiki](https://github.com/AdguardTeam/AdGuardHome/wiki) and [Discussions](https://github.com/AdguardTeam/AdGuardHome/discussions) and found no answer

- [X] I have searched other issues and found no duplicates

- [X] I want to request a feature or enhancement and not ask a question

### The problem

Currently, under DHCP Settings, if you click the "Reset all leases" button, it will clear out the dynamic `DHCP Leases` as well as `DHCP Static Leases`.

### Proposed solution

Add a separate button for `DHCP Leases` to clear out just the dynamic leases, preserving static leases.

### Alternatives considered and additional information

Clear all leases and manually type/paste back in the static leases, which can be quite a chore if you have more than a dozen or two static leases.

**Comment by r0bb10** [comment]
i upvote this!

**Comment by Bellavista** [comment]
I vote for this addition. I have 20 dynamic IPs managed by DHCP, and two access points . When a phone with a hidden Mac changes its network access point, it blocks multiple IPs. To free up IPs, I need to reduce the lease time. It would be helpful to be able to delete IPs individually.

**Comment by ymac-login** [comment]
Yes to this, and yes to static leases configured in adguardhome.yaml!

**Comment by Sil3ntVip3r** [comment]
This appears to duplicate
[#2164](https://github.com/AdguardTeam/AdGuardHome/issues/2164): both request a
way to clear dynamic DHCP leases without deleting configured static leases.
Linking the older request for consolidation.
